Improve workspace packages (#1121)
OpenZeppelin/cairo-contracts: docs/modules/ROOT/pages/api/introspection.adoc

= Introspection

> This crate deals with https://en.wikipedia.org/wiki/Type_introspection[type introspection] of contracts, that is, examining which functions can be called on them. This is usually referred to as a contract’s interface.

Reviewer: ```suggestion
This crate handles https://en.wikipedia.org/wiki/Type_introspection[type introspection] of contracts. In other words, it examines which functions can be called on a given contract. This is referred to as the contract's interface.
```
